ovnimp.go

```diff
@@ -149,12 +149,23 @@ func (odbi *ovndb) transact(db string, ops ...libovsdb.Operation) ([]libovsdb.Op
 		return reply, err
 	}
 
-	if len(reply) < len(ops) {
-		for i, o := range reply {
-			if o.Error != "" && i < len(ops) {
-				return nil, fmt.Errorf("Transaction Failed due to an error : %v details: %v in %v", o.Error, o.Details, ops[i])
+	// Per RFC 7047 Section 4.1.3, the operation result array in the transact response object
+	// maps one-to-one with operations array in the transact request object. We need to check
+	// each of the operation result for null error to ensure that the transaction has succeeded.
+	for i, o := range reply {
+		if o.Error != "" {
+			// Per RFC 7047 Section 4.1.3, if all of the operations succeed, but the results
+			// cannot be committed, then "result" will have one more element than "params",
+			// with the additional element being an <error>.
+			opsInfo := "commiting transaction"
```

Comment:
Move this to constant either in this file or common.go

Reply:
@noah8713 done. PTAL